requirements.txt

```diff
@@ -7,13 +7,13 @@
 pyparsing<3,>=2.0.2
 csv-diff>=1.0
 loguru~=0.5
-imageio~=2.19
+imageio>=2.31.4
```

Comment:
It seems like we have one more error here:

```
E       AttributeError: module 'PIL.Image' has no attribute 'ANTIALIAS'
```

This is a bit strange, because I tested the changes from this branch in https://github.com/ivadomed/ivadomed/pull/1304, but did not encounter this error.

Nonetheless, I see that `Pillow<10.0.0` was pinned previously in this PR to fix this issue (https://github.com/ivadomed/ivadomed/pull/1297/commits/5672e73b11c9212790ebfd99ebae753cfe7e0056).

That said, I think we can fix this by applying the suggestion from https://stackoverflow.com/a/76616129 instead.